web/app.js

```diff
@@ -2102,10 +2102,7 @@ const PDFViewerApplication = {
   },
 
   unbindWindowEvents() {
-    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
-      throw new Error("Not implemented: unbindWindowEvents");
-    }
-    const { _boundEvents } = this;
+    const { _boundEvents, mainContainer } = this;
```

Comment:
This cannot possibly work, since there's no `PDFViewerApplication.mainContainer` property; I'd expect that manually calling `PDFViewerApplication.unbindWindowEvents` in the viewer would throw with the patch as-is.

```suggestion
    const { _boundEvents } = this;
    const { mainContainer } = this.appConfig;
```